### PR TITLE
add xpu api Chinese documents

### PR DESCRIPTION
**Pull request by vslyu**
add xpu api Chinese documents as follows:
- paddle.XPUPlace
image

- paddle.is_compiled_with_xpu
image

- paddle.static.xpu_places
image

### CODE REVIEW

File: doc/paddle/api/paddle/fluid/core/XPUPlace_cn.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+
+
+
+``XPUPlace`` 是一个设备描述符，表示一个分配或将要分配 ``Tensor`` 或 ``LoDTensor`` 的 Baidu Kunlun XPU 设备。
+每个 ``XPUPlace`` 有一个 ``dev_id`` （设备id）来表明当前的 ``XPUPlace`` 所代表的显卡编号，编号从 0 开始。
+``dev_id`` 不同的 ``XPUPlace`` 所对应的内存不可相互访问。
+这里编号指的是可见显卡的逻辑编号，而不是显卡实际的编号。
+可以通过 ``XPU_VISIBLE_DEVICES`` 环境变量限制程序能够使用的 Baidu Kunlun XPU 设备，程序启动时会遍历当前的可见设备，并从 0 开始为这些设备编号。
```
Comment (wangxicoding) [inline doc/paddle/api/paddle/fluid/core/XPUPlace_cn.rst]: 昆仑里这个目前不生效，先不写吧

Comment (vslyu) [reply doc/paddle/api/paddle/fluid/core/XPUPlace_cn.rst]: done, delete it.

**Review by wangxicoding — APPROVED** [review]
LGTM

File: doc/paddle/api/paddle/fluid/core/XPUPlace_cn.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+XPUPlace
+-------------------------------
+
+.. py:class:: paddle.XPUPlace
+
+
+
+
+
+``XPUPlace`` 是一个设备描述符，表示一个分配或将要分配 ``Tensor`` 或 ``LoDTensor`` 的 Baidu Kunlun XPU 设备。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/core/XPUPlace_cn.rst]: 这里应该不需要写  ``LodTensor`` , 统一写Tensor 即可

File: doc/paddle/api/paddle/fluid/framework/is_compiled_with_xpu_cn.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+-------------------------------
+
+.. py:function:: paddle.is_compiled_with_xpu()
+
+
+
+
+检查 ``whl`` 包是否可以被用来在Baidu Kunlun XPU上运行模型
+
+返回：支持Baidu Kunlun XPU则为True，否则为False。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/framework/is_compiled_with_xpu_cn.rst]: 返回：bool，支持Baidu Kunlun XPU则为True，否则为False。 下面15行的返回删除

File: doc/paddle/api/paddle/fluid/core/XPUPlace_cn.rst (hunk 2 above)
Comment (vslyu) [reply doc/paddle/api/paddle/fluid/core/XPUPlace_cn.rst]: done.

File: doc/paddle/api/paddle/fluid/framework/is_compiled_with_xpu_cn.rst (hunk 3 above)
Comment (vslyu) [reply doc/paddle/api/paddle/fluid/framework/is_compiled_with_xpu_cn.rst]: done.

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**